components/Hits.js

```diff
@@ -7,13 +7,13 @@ class Hits extends React.Component {
   renderWithResults() {
     let renderedHits = map(this.props.results.hits, hit => {
       return (
-        <div className={this.props.cssClasses.item} key={hit.objectID}>
```

Comment:
Ah now I think I remember, the key should not be put on the Template directly but on a wrapping div, I think that's a react requirement.